core/errors/rewriter.h

```diff
@@ -17,5 +17,6 @@ constexpr ErrorClass InvalidStructMember{3511, StrictLevel::False};
 constexpr ErrorClass NilableUntyped{3512, StrictLevel::False};
 constexpr ErrorClass HasAttachedClassInClass{3513, StrictLevel::False};
 constexpr ErrorClass ContravariantHasAttachedClass{3514, StrictLevel::False};
+constexpr ErrorClass InvalidChainedSig{3515, StrictLevel::False};
```

Comment:
I am not sure whether it would be better to split this error code up or not.

Reply:
I think the idea was to keep one error per rewriter. Others seem to be similar but we are raising more than them (6 instances).